modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -4,13 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, esb, deploy, architecture
 
-A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+A Mule project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
 
-* Running multiple applications side-by-side
-* Clear boundaries for operations on what a Mule application is
-* Mule can monitor your applications and will reload configuration changes automatically
-* Applications can depend on different library versions, even if they would conflict before
-* Multiple versions of an application can exist side by side
+* Running independently multiple applications side-by-side. You can include the same name spaces within different applications and they neither collide nor share information.
+* Clear boundaries for operations on what a Mule application is.
+* Mule can monitor your applications and will reload configuration changes automatically.
+* Applications can depend on different library versions, even if they would conflict before.
+* Multiple versions of an application can run side-by-side within the same Mule container.
```

Comment:
I think you can remove "side-by-side" from the sentence.

Reply:
Removed.